src/pages/ItemForm/index.tsx

```diff
@@ -315,12 +269,21 @@ export default function ItemForm() {
         headers: { "Content-Type": "application/json" },
         body: JSON.stringify(data),
       });
+      if (response) {
+        alert("Issue is successfully created");
```

Comment:
please don't use the browser alert, use the ribbon that you were using

Reply:
Changed it.